robot/trajectory_generator.py

```diff
@@ -0,0 +1,67 @@
+import os
+import pickle
+import pathfinder as pf
+import wpilib
+
+
+WHEELBASE_WIDTH = 2 # In feet
+TRAJECTORY_DIRECTORY = 'trajectories'
+
+
+trajectories = {
```

Comment:
Is there a better way to do this then listing them in `trajectory_generator`? This feels like something that would fit better in autonomous files. @ArchdukeTim may have better intuition on this.

Reply:
You could read them in from a JSON? Pathfinder will also read in directly from a pathfinder binary created by he `serialize` methods, and you can also do a CSV. I think the best alternative  would be to store them all in a JSON file, but I don’t think it’s necessarily bad to do it here